.circleci: Cleanup unused ecr gc files

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #62286

ECR GC has been migrated to GHA we should remove these files now as they
are no longer used

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29943581](https://our.internmc.facebook.com/intern/diff/D29943581)

cc @seemethere @malfet @pytorch/pytorch-dev-infra

seemethere committed:
.circleci: Cleanup unused ecr gc files

ECR GC has been migrated to GHA we should remove these files now as they
are no longer used

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29943581).

malfet reviewed (approved):
Deleted code is tested code!

seemethere committed:
Update on ".circleci: Cleanup unused ecr gc files"


ECR GC has been migrated to GHA we should remove these files now as they
are no longer used

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29943581](https://our.internmc.facebook.com/intern/diff/D29943581)

[ghstack-poisoned]

seemethere commented:
@seemethere has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29943581).

seemethere committed:
Update on ".circleci: Cleanup unused ecr gc files"


ECR GC has been migrated to GHA we should remove these files now as they
are no longer used

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D29943581](https://our.internmc.facebook.com/intern/diff/D29943581)

[ghstack-poisoned]
